src/lib/Form/DataTransformer/DateTimeFormatTransformer.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+use EzSystems\EzPlatformAdminUi\Service\UserPreferences\DateTimeFormat\DateTimeFormatTransformer as Transformer;
+
+class DateTimeFormatTransformer implements DataTransformerInterface
+{
+    /**
+     * @var \EzSystems\EzPlatformAdminUi\Form\DataTransformer\DateTimeFormatTransformer
+     */
+    private $transformer;
```

Comment:
Naming is very confusing here. I'd rename `\EzSystems\EzPlatformAdminUi\Form\DataTransformer\DateTimeFormatTransformer` to avoid confusion with Form DataTransformers.